test/integration/http_protocol_integration.h

```diff
@@ -58,6 +58,80 @@ class HttpProtocolIntegrationTest : public testing::TestWithParam<HttpProtocolTe
     setDownstreamProtocol(GetParam().downstream_protocol);
     setUpstreamProtocol(GetParam().upstream_protocol);
   }
+
+protected:
+  void expectUpstreamWireBytesSentAndReceived(std::string log_name, int log_index,
```

Comment:
Consider cleaning up the arguments to this functions -- the list is very long making it hard to read. Is there a reason we'd have both H1 wire bytes send/received and also H2 wire bytes sent/received in one invocation? I'd expect it to be mutually exclusive.

Reply:
Will group the h1 and h2 expectation into 2 vectors.